expressEngine/customer/service/customer.js

```diff
@@ -31,6 +32,16 @@ class Service extends BaseService {
             return this.error(err);
         }
     }
+    delete = async (data) => {
+        const { id } = data.params;
+        let valid = mongoose.Types.ObjectId.isValid(id);
```

Comment:
@rahuljainss Create a helper method to check it.

Reply:
Sir done with the changes please check.